Source/AviationWeather.Test/Build/AviationWeather.Test.vcxproj

```diff
@@ -21,6 +20,7 @@
   </ItemGroup>
   <PropertyGroup>
     <LinkIncremental>true</LinkIncremental>
+    <CodeAnalysisRuleSet>NativeRecommendedRules.ruleset</CodeAnalysisRuleSet>
```

Comment:
It's a rule set for code analysis specific to managed projects. I think the migration to VS2015 Update 2 inserted it. Removed.